populations/__init__.py

```diff
@@ -75,6 +74,11 @@ def from_samples(label, samples, param_dict, sensitivity=None, \
         store_optimal_snrs : bool
             whether to store optimal SNRs for each sample (only used if mock uncertainty is SNR-dependent)
         """
+
+        max_samps=kwargs['max_samps']
```

Comment:
can you change these and the following lines to something like: 
`max_samps = kwargs['max_samps'] if 'max_samps' in kwargs else _max_samps_default`